src/components/ProtectedRoute.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Navigate } from 'react-router-dom';
+import { getAccessToken } from '../util/auth';
+
+const ProtectedRoute = ({ children }) => {
+  const isAuthenticated = getAccessToken();
```

Comment:
```suggestion
 const hasAccessToken = getAccessToken();
```

Your helper returns token. So this naming would be more explicit